### PR TITLE
Add 13 dev release back to the site

### PR DESCRIPTION
**Pull request by karesti**
put dev 13 in the website

### CODE REVIEW

File: _data/ispn.yml [hunk 1]
```diff
@@ -54,47 +54,46 @@ stable:
...
+    dev: /docs/dev/titles/developing/developing.html
+    upgrade: /docs/dev/titles/upgrading/upgrading.html
+    config: /docs/dev/titles/configuring/configuring.html
+    tuning: /docs/dev/titles/tuning/tuning.html
+    spring: /docs/stable/titles/spring/spring.html
+    hibernate: /docs/stable/titles/hibernate/hibernate.html
+    starter: /docs/dev/titles/spring_boot/starter.html
+    home: /docs/dev/index.html
+    contributing: /docs/dev/titles/contributing/contributing.html
+    javadocs: https://docs.jboss.org/infinispan/12.1/apidocs/
```
Comment (oraNod) [inline _data/ispn.yml:94]:
```suggestion
    javadocs: https://docs.jboss.org/infinispan/13.0/apidocs/
```

File: _data/ispn.yml [hunk 2]
```diff
@@ -54,47 +54,46 @@ stable:
...
+    configdocs: https://docs.jboss.org/infinispan/12.1/configdocs/
+    jmxdocs: https://docs.jboss.org/infinispan/12.1/apidocs/jmxComponents.html
```
Comment (oraNod) [inline _data/ispn.yml:96]:
```suggestion
    configdocs: https://docs.jboss.org/infinispan/13.0/configdocs/
    jmxdocs: https://docs.jboss.org/infinispan/13.0/apidocs/jmxComponents.html
```

**Comment by karesti** [comment]
@oraNod rebased

**Review by oraNod — APPROVED** [review]

**Comment by oraNod** [comment]
thanks @karesti

**Pull request merged**